lib/pull.py

```diff
@@ -106,7 +106,7 @@ def download(self):
                # currently, this error is only raised if we’ve downloaded the
                # skinny manifest.
                have_skinny = True
-               if (ch.arch == "amd64"):
+               if (ch.arch in ["amd64", "aarch64"]):
```

Comment:
I think we talked about this? The issue was that your ARM VM was set up to silently execute x86-64 code via QEMU.